pulp_smash/utils.py

```diff
@@ -124,6 +125,22 @@ def get_importers(server_config, href, responses=None):
     ), responses)
 
 
+def get_repo(server_config, href, responses=None):
```

Comment:
This function seems to follow the same common pattern as other functions in this module. Can you add a new test case in module `tests.test_utils`? You should be able to create a class like `GetTestCase(CommonAssertionsMixin, TestCase)`.